config/cloud.cfg.tmpl

```diff
@@ -121,6 +155,10 @@ cloud_config_modules:
  - byobu
 {% endif %}
 
+{% if variant in ["photon"] %}
+unverified_modules: ['yum-add-repo', 'ntp']
```

Comment:
Since you added `photon` to the `distros` list in both of these modules, there should be no reason to add them as `unverified_modules`. This block should be deleted.